crypto/keyring/info.go

```diff
@@ -246,6 +232,7 @@ func (i multiInfo) UnpackInterfaces(unpacker codectypes.AnyUnpacker) error {
 
 // encoding info
 func marshalInfo(i Info) []byte {
+	// TODO: Why do we use Legacy Amino marshaling here?
 	return CryptoCdc.MustMarshalBinaryLengthPrefixed(i)
```

Comment:
can anyone answer if we can use protobuf here?

Reply:
This is related to #7108. We marshal the Info as amino, and amino bytes are stored in the keyring backend.